lib/gis/omp_threads.c

```diff
@@ -35,7 +44,7 @@ int G_set_omp_num_threads(struct Option *opt)
     G_verbose_message(_("%d threads are set up for parallel computing."),
                       threads);
 #else
-    if (threads != 1) {
+    if (threads != 0) {
```

Comment:
Should this test for 1 as well as before? If you set 1 it shouldn't complain about "only one thread will be used"

Reply:
You're right. I changed the logic to not print a warning on `0` or `1` if OpenMP is disabled.